web/adventurelookup/urls.py

```diff
@@ -13,12 +13,13 @@
     1. Import the include() function: from django.conf.urls import url, include
     2. Add a URL to urlpatterns:  url(r'^blog/', include('blog.urls'))
 """
-from django.conf.urls import url
+from django.conf.urls import url, include
 from django.contrib import admin
 from django.contrib.staticfiles.urls import staticfiles_urlpatterns
 
 urlpatterns = [
     url(r'^admin/', admin.site.urls),
+    url(r'^adventures/', include('adventures.urls'))
```

Comment:
Views should be namespaced to avoid name collision. You can do that by specifying the `namespace` parameter to include:
`include('adventures.urls', namespace='adventures')`